### PR TITLE
add_flow_to_list_folder

### PR DESCRIPTION
**Pull request by sadiqueali-a9011**

### CODE REVIEW

File: grid/src/List/ListItem.js [hunk 1]
```diff
@@ -33,7 +33,37 @@ const ListItem = ({
     rowSelector,
     rowActions,
     expandableColumn
-}) => {
+}: {
+    row: Object,
+    idAttribute: String,
+    style: Object,
+    theme: string,
```
Comment (iCargopoc) [inline grid/src/List/ListItem.js:40]: Capital S is needed for String - I guess. Please check line 38.

File: grid/src/List/ParentItem.js [hunk 2]
```diff
@@ -15,16 +15,16 @@ const ParentItem = ({
     isParentRowOpen,
```
Comment (iCargopoc) [inline grid/src/List/ParentItem.js:15]: I think we need to give types of props in this component.

**Review by iCargopoc — CHANGES_REQUESTED** [review]

File: grid/src/List/ListItem.js (hunk 1 above)
Comment (sadiqueali-a9011) [reply grid/src/List/ListItem.js:40]: Yes thats correct, when we use Cap String some type checking in the code  are not compatible with String.
example :  theme: String , ( not compatible )

 if (theme === "portal") {
    rowItemHeight += 10;
}

Inorder to resolve this we have used small string.

File: grid/src/List/ParentItem.js (hunk 2 above)
Comment (sadiqueali-a9011) [reply grid/src/List/ParentItem.js:15]: I have added props types

**Comment by iCargopoc** [comment]
Closing as the new one is created

**Pull request closed without merging**